src/collection-browser.ts

```diff
@@ -658,6 +667,11 @@ export class CollectionBrowser
     `;
   }
 
+  private toggleManageViewRemoveButtonState() {
```

Comment:
Suggestion: method name: This doesn't "toggle" the button state, it more precisely "updates" it.

Suggest replace state variable with method
```
\\ ?enableRemoveButton=${this.manageViewRemoveButtonEnable}
?enableRemoveButton=${this.anySelected}
...
get anySelected(): boolean {
  return this.dataSource.checkedTileModels.length !== 0;
}
```